packages/react/src/Pivot.ts

```diff
@@ -1 +0,0 @@
-export * from '@fluentui/react-tabs/lib/Pivot';
```

Comment:
we should probably add `Tab` but keeping `Pivot` as is and deprecate. let's follow up tmr

Reply:
I'm planning to export the `Pivot` component from Tabs.ts as a deprecated alias of `Tabs`, in Part 2. As of this PR, the component itself is still named `Pivot`.

If there's a reason to keep the _file_ Pivot.ts around, I can add it back. I suppose it would help avoid problems with places that import from the Pivot.ts file directly.

Reply:
We should keep `packages/react/src/Pivot.ts` since we consider that part of the public API that people are allowed to import directly from (maybe also `packages/react-tabs/src/Pivot.ts`).